### PR TITLE
chore: legacy URL migration

### PR DESCRIPTION
**Pull request by mfw78**
# Description

🚨 **DO NOT MERGE THIS PR** 🚨

This PR provides the URLs and process scripts that were used to generate the `vercel.json` for the redirects, and includes the CVS from which these were generated from. This serves only to document the migration purposes so that the process can be checked for merging #273.

# Process

1. Extract the `404.csv` from the Google document that was being worked on.
2. Run `process.js` script over the `404.csv` which generates a `vercel2.json`.
3. Copy the `vercel2.json` contents into `vercel.json`.
4. Push the PR (see #273)
5. As the previous PR is pushed now, use the `process.js` script (do `yarn` to install `node-fetch` deps from this PR) to verify the correct redirects.

🚨 Reminder: **DO NOT MERGE THIS PR** 🚨

### CODE REVIEW

File: check.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
```
Comment (anxolin) [inline check.js:1]: This is great. probably better would be to make this a unit test

This way, we can run it as a Github Action. In the future can be evolved to be a merge criteria.

I would foresee we will end up creating a git hook that checks moved MD files and ensures everything there is in the CSV before we commit. If we have the test that checks that everything in the CSV is in the redirect, then we make sure is impossible to make 404s if we move folders around

**Review by anxolin — APPROVED** [review]
Worked like a charm!

image

**Comment by mfw78** [comment]
@alfetopito I do applaud your strong desire to keep things clean :broom: :soap: though please leave this branch here as it serves as repository documentation for how #273 was generated :pray:

**Comment by alfetopito** [comment]
> @alfetopito I do applaud your strong desire to keep things clean 🧹 🧼 though please leave this branch here as it serves as repository documentation for how #273 was generated 🙏

Oops. Sorry about that.

**Pull request closed without merging**